diff --git a/Telegram/lib_ui/ui/widgets/fields/input_field.cpp b/ui/widgets/fields/input_field.cpp
index 7f3c6b4..074bff8 100644
--- a/Telegram/lib_ui/ui/widgets/fields/input_field.cpp
+++ b/Telegram/lib_ui/ui/widgets/fields/input_field.cpp
@@ -3945,12 +3945,7 @@ void InputField::keyPressEventInner(QKeyEvent *e) {
 		_reverseMarkdownReplacement = false;
 	}
 
-	if (backspace && macmeta) {
-		QTextCursor tc(textCursor()), start(tc);
-		start.movePosition(QTextCursor::StartOfLine);
-		tc.setPosition(start.position(), QTextCursor::KeepAnchor);
-		tc.removeSelectedText();
-	} else if (backspace
+	if (backspace
 		&& e->modifiers() == 0
 		&& revertFormatReplace()) {
 		e->accept();
