diff --git a/themes/fletcherm.zsh-theme b/themes/fletcherm.zsh-theme
index e9618854..a296f93b 100644
--- a/themes/fletcherm.zsh-theme
+++ b/themes/fletcherm.zsh-theme
@@ -1,5 +1,5 @@
 # Copied from old version of tonotdo's theme. LSCOLORS modified.
-PROMPT='%{$fg_no_bold[cyan]%}%n%{$fg_no_bold[magenta]%}•%{$fg_no_bold[green]%}%3~$(git_prompt_info)%{$reset_color%}» '
+PROMPT='%{$fg_no_bold[cyan]%}%n@%m%{$fg_no_bold[magenta]%}•%{$fg_no_bold[green]%}%3~$(git_prompt_info)%{$reset_color%}» '
 RPROMPT='[%*]'
 
 # git theming
